playbook/app/pb_kits/playbook/pb_date_time_stacked/docs/_date_time_stacked_default.jsx

```diff
@@ -9,17 +9,20 @@ const DateTimeStackedDefault = (props) => (
         {...props}
     />
     <br />
+    <DateTimeStacked
+        datetime={new Date(2018, 2, 20)}
+        {...props}
+    />
```

Comment:
Curious to know why we're adding another example to the default here.

Reply:
To make it match the Rails doc examples